src/segmented-control/__tests__/desktop.test.tsx

```diff
@@ -1,14 +1,15 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import React from 'react';
+import { KeyCode } from '@cloudscape-design/test-utils-core/dist/utils';
```

Comment:
This should be `@cloudscape-design/test-utils-core/utils`

```suggestion
import { KeyCode } from '@cloudscape-design/test-utils-core/utils';
```

Reply:
Good catch, updated.